src/parser/tests.rs

```diff
@@ -37,13 +37,15 @@ const VALID_SCRIPTS: &'static [&'static str] = &[
 
     // Predicates
     "wasm:bytecode:br:before / i / { }",
-    "wasm:bytecode:br:before / \"i\" <= 1 / { }",
+    r#"wasm:bytecode:br:before / "i" <= 1 / { }"#,  // Alex How is this a valid script
```

Comment:
To answer comment: This is checking syntax rather than semantics. So, the type checker (when we have it) should fail on this script, but the parser shouldn't.